src/BootcampMauiApp/AppShell.xaml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Shell
+    x:Class="BootcampMauiApp.AppShell"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:local="clr-namespace:BootcampMauiApp"
+    xmlns:pages="clr-namespace:BootcampMauiApp.Pages"
+    Title="BootcampMauiApp">
+
+    <TabBar>
```

Comment:
Can you use the flyout instead as tabs are ugly. Also, use icons. 

Here is docs on how to add images to maui: https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/images?view=net-maui-9.0